### PR TITLE
CTOR- 938 : Refresh cloud-azure-management-monitor

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh cloud-azure-management-monitor

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md [hunk 1]
```diff
@@ -2,127 +2,407 @@
 id: cloud-azure-management-monitor
 title: Azure Monitor
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Prerequisites
+## Dépendances du Connecteur de supervision
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md:8]:
```suggestion
## Dépendances du connecteur de supervision
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md [hunk 2]
```diff
@@ -2,127 +2,407 @@
...
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="Cloud-Azure-Management-Monitor-custom" label="Cloud-Azure-Management-Monitor-custom">
+
+| Alias       | Modèle de service                                           | Description                                           |
+|:------------|:------------------------------------------------------------|:------------------------------------------------------|
+| Get-Metrics | Cloud-Azure-Management-Monitor-Get-Metrics-Api-custom       | Contrôle une ou plusieurs métriques d'une ressource   |
+| Kusto-Query | Cloud-Azure-Management-Log-Analytics-Kusto-Query-Api-custom | Execute une requête KQL/Kusto via l'API Log Analytics |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md:31]:
```suggestion
| Kusto-Query | Cloud-Azure-Management-Log-Analytics-Kusto-Query-Api-custom | Exécute une requête KQL/Kusto via l'API Log Analytics |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md [hunk 3]
```diff
@@ -2,127 +2,407 @@
...
+
+> La liste et les descriptions des métriques disponibles pour les ressources Azure est [ici](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/metrics-index).
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md:33]:
```suggestion
> Retrouvez [ici la liste et les descriptions des métriques disponibles pour les ressources Azure](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/metrics-index).
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md [hunk 4]
```diff
@@ -2,127 +2,407 @@
...
+</Tabs>
+
+## Utiliser le connecteur de supervision
+
+### Utiliser un modèle d'hôte issu du connecteur
+
+1. Ajoutez un hôte à Centreon depuis la page **Configuration > Hôtes**.
+2. Remplissez le champ **Adresse IP/DNS** avec l'adresse **127.0.0.1**.
+3. Appliquez le modèle d'hôte **Cloud-Azure-Management-Monitor-custom**. Une liste de macros apparaît. Les macros vous permettent de définir comment le connecteur se connectera à la ressource, ainsi que de personnaliser le comportement du connecteur.
+4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires. Par exemple, pour ce connecteur, **AZURECUSTOMMODE** (valeurs possibles : **api** ou **azcli**). En effet, il existe plusieurs modes de communication avec l'équipement supervisé : soit l'outil en ligne de commande azcli, soit une interrogation directe de l'api.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md:161]:
```suggestion
4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires. Par exemple, pour ce connecteur, **AZURECUSTOMMODE** (valeurs possibles : **api** ou **azcli**). En effet, il existe plusieurs modes de communication avec l'équipement supervisé : soit l'outil en ligne de commande azcli, soit une interrogation directe de l'API.
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md [hunk 5]
```diff
@@ -2,127 +2,406 @@
...
+
+<Tabs groupId="sync">
+<TabItem value="Cloud-Azure-Management-Monitor-custom" label="Cloud-Azure-Management-Monitor-custom">
+
+| Service Alias | Service Template                                            | Service Description                        |
+|:--------------|:------------------------------------------------------------|:-------------------------------------------|
+| Get-Metrics   | Cloud-Azure-Management-Monitor-Get-Metrics-Api-custom       | Check one or several metrics of a resource |
+| Kusto-Query   | Cloud-Azure-Management-Log-Analytics-Kusto-Query-Api-custom | Query Log Analytics API using KQL syntax   |
+
+> The list and descriptions of metrics available for Azure resources are [here](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/metrics-index).
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md:32]:
```suggestion
> Find [here the list and descriptions of metrics available for Azure resources](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/metrics-index).
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md [hunk 6]
```diff
@@ -2,127 +2,406 @@
...
+</Tabs>
+
+### Collected metrics & status
+
+Here is the list of services for this connector, detailing all metrics and statuses linked to each service.
+
+<Tabs groupId="sync">
+<TabItem value="Get-Metrics" label="Get-Metrics">
+
+| Name   | Unit  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md:45]: doublon

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md [hunk 7]
```diff
@@ -2,127 +2,407 @@
...
+### Métriques & statuts collectés
+
+Voici le tableau des services pour ce connecteur, détaillant les métriques et statuts rattachés à chaque service.
+
+<Tabs groupId="sync">
+<TabItem value="Get-Metrics" label="Get-Metrics">
+
+| Nom    | Unité |
+|:-------|:------|
+| metric | N/A   |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md:48]: doublon

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md [hunk 8]
```diff
@@ -2,127 +2,407 @@
...
+| metric | N/A   |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md:49]:
```suggestion
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md [hunk 9]
```diff
@@ -2,127 +2,406 @@
...
+|:-------|:------|
+| metric | N/A   |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-azure-management-monitor.md:47]:
```suggestion
```

**Pull request merged**